en/08_Changelogs/6.0.0.md

```diff
@@ -1259,6 +1256,10 @@ This is done to help developers debug validation issues that happen outside of a
 
 The value of the field that failed validation is intentially not included in the output, as it could contain sensitive information e.g. an API token.
 
+### `silverstripe/campaign-admin` no longer part of `silverstripe/recipe-cms` {#campaign-admin}
+
+The `silverstripe/campaign-admin` module is no longer included in `silverstripe/recipe-cms`. If your `composer.json` requires `silverstripe/recipe-cms` and you wish to continue using `silverstripe/campaign-admin`, then you will need to add manually add to your project via `composer.json`.
```

Comment:
```suggestion
The `silverstripe/campaign-admin` module is no longer included in `silverstripe/recipe-cms`. If your `composer.json` requires `silverstripe/recipe-cms` and you wish to continue using `silverstripe/campaign-admin`, then you will need to add manually add to your project's `composer.json`.
```
Saying add "via" `composer.json` could imply there's some magic that prevents the CLI from adding it correctly.

Reply:
Updated